### DISCUSSION TITLE
Support for Yarn feasible?

### DISCUSSION THREAD

**Discussion by knutwannheden** [caoccao/Javet#287] [Q&A] [answered]
I couldn't find Yarn mentioned anywhere, so I assume Javet has no direct support for it. I suppose it should all work when Yarn is configured to use a `node_modules` layout, but won't work with the standard PnP layout. (See https://yarnpkg.com/features/pnp for details.)

Now to my question: How hard would it be to support additional package managers like Yarn in Javet?

**Comment by caoccao** [comment, marked as answer]
Javet works with the traditional yarn seamlessly. That might be the reason why nobody raises any issues on yarn.

Could you leave a minimal yarn pnp project for me to reproduce the issue?

**Reply by knutwannheden** [reply]
Thanks for your reply. So far I am just checking what the capabilities are. If I run into problems using Yarn I will create an issue for that.
